CHANGELOG-zh-CN.md

```diff
@@ -105,6 +105,72 @@ SQL 开发
 
 ### 缺陷修复
 
+连接会话
+
+- 部分场景会话在过期后未能删除，潜在资源泄漏风险 (#2125)
+- 在高负载下 SingleConnectionDataSource.getConnection() 可能会出现并发问题 (#1914)
+- 在数据源配置修改用户名大小写之后，连接 OceanBase Oracle 可能会出错 (#1797)
+- 打开 SQL 控制台时偶先 404 错误 (#1809)
+
+SQL 执行
+
+- OceanBase v4.2 提交/回滚按钮状态与实际事务状态不同步 (#2097)
+- 带有单行注释的 SQL 语句执行失败 (#2085)
+- 最后一个 SQL 命令若不以分隔符结束将导致偏移量计算错误 (#1970)
+- 匿名块执行与 Oracle 11g 不兼容 (#1759)
+- 时间戳类型显示时精度丢失 (#1411)
+
+结果集
+
+- 结果集中同时修改多行数据时耗时较久 (#2007)
+- Timestamp 类型显示时精度丢失 (#1996)
+
+表对象
+
+- 索引和约束视图中列名顺序不一致 (#1948)
+- 无法查看 MySQL v5.6 表详情 (#1635)
+- 无法查看 Sofa ODP 表详情 (#2043)
+
+无锁结构变更
+
+- 当输入语句包含注释时，OSC 任务会因为语法异常而失败 (#1597)
+
+SQL检查
+
+- 存在虚拟列时发生空指针异常 (#2031)
+- 通过 DROP CONSTRAINT 删除主键的操作未能检测到 (#1879)
+- 特定 ALTER 语句会触发的空指针异常 (#1865)
+
+分区计划
+
+- 当模式或表名为小写时，无法执行分区 DDL (#2088)
+- 当分区上界非顺序地按年/月/日增长时，分区生成错误 (#1992)
```

Comment:
This is a problem exposed during testing and is not perceived by users.

Reply:
removed